src/main/java/me/zeroeightsix/kami/module/modules/misc/HighwayTools.kt

```diff
@@ -0,0 +1,738 @@
+    private val noViewReset = register(Settings.booleanBuilder("NoViewReset").withValue(true).withVisibility { page.value == Page.MAIN }.build())
+    private val rotate = register(Settings.booleanBuilder("Rotate").withValue(true).withVisibility { page.value == Page.MAIN }.build())
+    private val stats = register(Settings.booleanBuilder("ShowStats").withValue(true).withVisibility { page.value == Page.MAIN }.build())
+    private val filled = register(Settings.booleanBuilder("Filled").withValue(true).withVisibility { page.value == Page.MAIN }.build())
+    private val outline = register(Settings.booleanBuilder("Outline").withValue(true).withVisibility { page.value == Page.MAIN }.build())
+    private val aFilled = register(Settings.integerBuilder("Filled Alpha").withMinimum(0).withValue(31).withMaximum(255).withVisibility { filled.value && page.value == Page.MAIN }.build())
+    private val aOutline = register(Settings.integerBuilder("Outline Alpha").withMinimum(0).withValue(127).withMaximum(255).withVisibility { outline.value && page.value == Page.MAIN }.build())
+
+    //Custom build settings
+    val clearSpace = register(Settings.booleanBuilder("ClearSpace").withValue(true).withVisibility { page.value == Page.BUILD }.build())
+    var clearHeight = register(Settings.integerBuilder("ClearHeight").withMinimum(1).withValue(4).withMaximum(6).withVisibility { page.value == Page.BUILD && clearSpace.value }.build())
+    private var buildWidth = register(Settings.integerBuilder("BuildWidth").withMinimum(1).withValue(7).withMaximum(9).withVisibility { page.value == Page.BUILD }.build())
+    private val rims = register(Settings.booleanBuilder("Rims").withValue(true).withVisibility { page.value == Page.BUILD }.build())
+    private var rimHeight = register(Settings.integerBuilder("RimHeight").withMinimum(1).withValue(1).withMaximum(clearHeight.value).withVisibility { rims.value && page.value == Page.BUILD }.build())
+    private val cornerBlock = register(Settings.booleanBuilder("CornerBlock").withValue(true).withVisibility { page.value == Page.BUILD }.build())
+
+    var ignoreBlocks = mutableListOf(ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "standing_sign")),
+            ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "wall_sign")),
+            ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "standing_banner")),
+            ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "wall_banner")),
+            ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "bedrock")),
+            ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "portal")))
+    var material: Block = ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "obsidian"))!!
+    var fillerMat: Block = ForgeRegistries.BLOCKS.getValue(ResourceLocation("minecraft", "netherrack"))!!
```

Comment:
shouldn't we have a better way of soring these? Why don't we just store a String of `minecraft:block-name`?